workspaces/liferay-sample-workspace/language/README.markdown

```diff
@@ -0,0 +1,6 @@
+To import translations into Liferay's Language Override tool, place them into a `[Workspace Root]/language/Language.properties` file using `key=value` syntax and run `../gradlew :buildLang` from the `language` folder. Deploy the translations by running `../gradlew deploy`. Translations with _(Automatic Copy)_ at the end are not imported into Liferay.
```

Comment:
I'd recommend moving "Deploy the translations by running `../gradlew deploy`. Translations with _(Automatic Copy)_ at the end are not imported into Liferay." to it's own paragraph.

I'm also guessing that Brian would prefer to not have the Related Documentation section since it would be a pain to update those URLs when they change on learn.